saturday-afternoon/utils/resolveScopedStyles.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as React from "react";
+
+export default function resolveScopedStyles(scope: React.Element<any>) {
```

Comment:
do we really need this function? adds some complexity to the code

Reply:
the less we need to explain to attendants the better

Reply:
Yes, let's make it as simple as possible